src/main/kotlin/baseball/domain/PlayerBall.kt

```diff
@@ -0,0 +1,30 @@
+package baseball.domain
+
+private const val BALL_SIZE = 1
+private const val VALID_BALLS_SIZE = 3
+private const val NUMBER_REGEX = "\\d"
+
+class PlayerBall(numbers: String) {
+
+    val numbers: List<Int>
+
+    init {
+        val balls: List<String> = numbers.chunked(BALL_SIZE)
+        validateBalls(balls)
+
+        this.numbers = balls.map { ball -> ball.toInt() }
+    }
+
+    private fun validateBalls(balls: List<String>) {
+        if (!isNumberBall(balls)) {
+            throw IllegalArgumentException("숫자만 입력할 수 있습니다.")
+        }
+        if (balls.size != VALID_BALLS_SIZE) {
+            throw IllegalArgumentException("숫자는 3개를 입력해야 합니다.")
+        }
+    }
+
+    private fun isNumberBall(balls: List<String>): Boolean {
+        return balls.all { ball -> ball.matches(NUMBER_REGEX.toRegex()) }
+    }
```

Comment:
아래 방식처럼 사용할 수도 있겠군요!
```suggestion
    private fun isNumberBall(balls: List<String>): Boolean {
        return balls.all { it.toIntOrNull() != null }
    }
```

아니면
```kotlin
require(balls.all { it.toIntOrNull() != null }) { "숫자만 입력할 수 있습니다." }
```

Reply:
와 미쳤어요!